spec/services/sign_in/session_creator_spec.rb

```diff
@@ -118,7 +118,7 @@
           session = subject.session
           expect(session.handle).to eq(expected_handle)
           expect(session.hashed_refresh_token).to eq(expected_double_hashed_parent_refresh_token)
-          expect(session.refresh_creation).to eq(expected_created_time)
+          expect(session.refresh_creation.round(3)).to eq(expected_created_time)
```

Comment:
do we need to `round` here if we did `round` on the `expected_created_time`?

Reply:
Oooh I just realized you stubbed Time.now in the `before`...

`allow(Time.zone).to receive(:now).and_return(expected_created_time)`

So yes, then it shouldn't be necessary. I already kicked off a test in the pipeline on another branch to confirm, but I think you're right.

Reply:
Fixed :)

Reply:
went ahead and did this ✅